ocaml/typing/predef.mli

```diff
@@ -78,6 +78,13 @@ val build_initial_env:
   (Ident.t -> extension_constructor -> 'a -> 'a) ->
   'a -> 'a * 'a
 
+(* Add simd types to an environment.  We can't do this in [build_initial_env]
```

Comment:
This comment smells a bit stale.... it's not added in `build_initial_env`, but pretty soon thereafter.

Reply:
Adjusted - see if you like it.